src/Utils/Utils.php

```diff
@@ -158,24 +158,31 @@ public static function getCleanShopwareVersion(): string
      */
     protected static function getShopwareVersion(): string
     {
+        $version = null;
         // Composer 2 runtime API uses the `InstalledVersions::class` in favor of the
         // deprecated/removed `Versions::class`
         if (class_exists(InstalledVersions::class)) {
-            $platformIsInstalled = InstalledVersions::isInstalled('shopware/platform');
-            if ($platformIsInstalled) {
-                return InstalledVersions::getPrettyVersion('shopware/platform');
+            if (InstalledVersions::isInstalled('shopware/platform')) {
+                $version = $version ?: InstalledVersions::getPrettyVersion('shopware/platform');
```

Comment:
Since PHP 7.4 you can use `??=`, however since Shopware 6.2 is based on PHP 7.2/7.3, we can only use `??`:

PHP >= 7.1
```suggestion
                $version = $version ?? InstalledVersions::getPrettyVersion('shopware/platform');
```

PHP >= 7.4
```suggestion
                $version ??= InstalledVersions::getPrettyVersion('shopware/platform');
```